recipes/xtrabackuprb.rb

```diff
@@ -1,24 +1,25 @@
 include_recipe 'git'
 include_recipe 'percona::package_repo'
 
+package 'percona-xtrabackup'
```

Comment:
Above here you need to add ``include_recipe 'yum-epel' if node['platform_version'].to_i == 6``